pandas/tests/frame/test_alter_axes.py

```diff
@@ -41,49 +40,76 @@ def test_convert_dti_to_series(self):
             ),
             name="B",
         )
+        assert expected.dtype == idx.dtype
```

Comment:
I'm probably a little rusty on review :-) but is using assert in a fixture a common idiom?

Reply:
its probably not super-encouraged but i dont think its an anti-pattern